packages/views.api.unstable/src/manifest.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2024 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type {
+  EventDefinition,
+  ParameterConfig,
+  ParameterDefinition,
+} from "./config.js";
+
+export interface ViewManifest {
```

Comment:
something like this?
```suggestion
export interface ViewManifestV1 {
  version: "v1"
```

Also, where do you think is the best place for us to formally document this v1 spec, should it live on public docs? I am assuming we want to say as long as you produce this format in the given location you're compatible even if you don't want to use the vite plugin and/or vite

Reply:
oops i forgot the version, my bad 🤦‍♀️ 
i think public docs is probably best, yeah